hathor/p2p/states/ready.py

```diff
@@ -180,3 +206,32 @@ def handle_pong(self, payload: str) -> None:
         self.ping_min_rtt = min(self.ping_min_rtt, self.ping_rtt)
         self.ping_start_time = None
         self.log.debug('rtt updated', rtt=self.ping_rtt, min_rtt=self.ping_min_rtt)
+
+    def send_get_blocks(self) -> None:
+        """ Send a GET-BLOCKS command, requesting a list of latest N blocks from the best blockchain.
+        """
+        self.send_message(ProtocolMessages.GET_BLOCKS, '10')
+
+    def handle_get_blocks(self, payload: str) -> None:
+        """ Executed when a GET-BLOCKS command is received. It just responds with
+        a list of the latest N blocks from the best blockchain.
+        """
+        # TODO: implement self.get_best_blockchain_blocks(payload) 
+        n = int(payload)
+        assert n > 0
+        assert n < 100
+
+        block_tips = self.protocol.connections.manager.tx_storage.get_block_tips();
```

Comment:
The tips will technically return a list, because we still accept best block ties. I think we shouldn't expose block ties and instead maybe use the height index, which is free from ties, maybe the even add a `get_best_n_blocks` to the height index and use that.

Reply:
Improved.